### PR TITLE
New Port: Penguin Command

### PR DESCRIPTION
**Pull request by Cebion**
Game Information
Title: Penguin Command
URL: https://www.linux-games.com/penguin-command/

Controls:
| Button | Action |
|--|--| 
|Analogue Sticks| Move Mouse|
|Start | Start Game / Left mouse button|
|A| Right Cannon|
|B| Middle Cannon|
|X| Left Cannon|

CFW Tests:
[X] AmberELEC
[X] ArkOS
[X] ROCKNIX
[X] MuOS
[] Knulli (Optional)
[] Crossmix (Optional)

Architectures: 
[x] Aarch64
[X] Panfrost (Mainline)

Resolutions: 
[] 480x320 (Optional)
[X] 640x480
[X] 720x720 (RGB30) (Optional)
[X] Higher resolutions (e.g., 1280x720) 

Ready to Run!

### CODE REVIEW

File: ports/penguincommand/screenshot.png
Comment (cdeletre) [inline ports/penguincommand/screenshot.png:1]: The image might be smaller in jpg but a bit blurred. It's up to you :)
screenshot

Comment (Cebion) [reply ports/penguincommand/screenshot.png:1]: > The image might be smaller in jpg but a bit blurred. It's up to you :) screenshot

took it on device and put it trough your png thingo.

Comment (cdeletre) [reply ports/penguincommand/screenshot.png:1]: uh ? Not sure we understood each other :) I was saying that it's possible to get a smaller file size in jpg format (something like 60 KB)

Comment (JanTrueno) [reply ports/penguincommand/screenshot.png:1]: Ew, jpg artifacts ;)

**Pull request merged**